app/src/main/java/com/brewfind/MenuPageActivity.kt

```diff
@@ -0,0 +1,11 @@
+package com.brewfind
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+
+class MenuPageActivity : AppCompatActivity() {
```

Comment:
Created an activity that connects with your menu page layout. This way you should be able to setup button functionality from your main activity that takes you here.
Source: https://dev.to/carolmusyoka/how-to-start-another-activity-on-button-click-using-kotlin-50lk

Reply:
Ditto comments above.  This looks like an essentially empty class... I'm curious if it is reducing technical debt.  
The risk is:
1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which increases technical debt.